nepal_compliance/patches/add_custom_fields_v1.py

```
@@ -0,0 +1,5 @@
+import frappe
```

Ruff (0.12.2)

`frappe` imported but unused

Remove unused import: `frappe`

(F401)